KeyboardSpacer.js

```diff
@@ -1,89 +1,99 @@
 /**
  * Created by andrewhurst on 10/5/15.
  */
-var React = require('react');
-var ReactNative = require('react-native');
+import React, { Component, PropTypes } from 'react';
+import Keyboard from 'Keyboard';
```

Comment:
As of RN 0.27.2, this has changed again.  Keyboard should be imported from 'react-native'